console-frontend/src/components/hydra-console/HydraConsole.jsx

```diff
@@ -276,7 +291,54 @@ class HydraConsole extends React.Component {
         })
     }
 
+    setResourceID(name, value) {
+        this.getURL = true;
+
+        console.log(name +" "+ value)
+        let resourcesIDs = Object.assign({}, this.state.resourcesIDs);
+        resourcesIDs[name]['ResourceID'] = value.split('/').pop();
 
+        localStorage.setItem('resourceIDs', JSON.stringify(resourcesIDs))
+        
+        this.setState({
+            resourcesIDs: resourcesIDs
+        })
+    }
+
+    getEntryOutput(data) {
+        let arr = []
+        const classes = this.props.classes
+        data.map(entry => {
+            arr.push(<div className={classes.outputConsoleBraces}>&#123;</div>)
+
+            Object.keys(entry).map(key => {
+                arr.push(<div className={classes.outputEntryDiv}></div>)
+                    arr.push(<span className={classes.outputConsoleKey}>{key} :</span>)
+                    if(key == "@id")
+                    arr.push(<span className={classes.outputConsoleLink}
```

Comment:
You should consider adding {} to improve readability and futureproofing (in case we want to add something else)